crates/uv-extract/src/stream.rs

```diff
@@ -111,6 +111,18 @@ async fn untar_in<R: tokio::io::AsyncRead + Unpin, P: AsRef<Path>>(
     while let Some(entry) = pinned.next().await {
         // Unpack the file into the destination directory.
         let mut file = entry?;
+
+        // On Windows, skip symlink entries, as they're not supported. pip recursively copies the
+        // symlink target instead.
+        #[cfg(windows)]
```

Comment:
Can you make this `if cfg!(windows)` so the branch gets checked when compiling on unix?